sources/engine/Stride.Input/Windows/InputSourceWindowsDirectInput.cs

```diff
@@ -18,11 +20,15 @@ internal class InputSourceWindowsDirectInput : InputSourceBase
         private readonly HashSet<Guid> devicesToRemove = new HashSet<Guid>();
         private InputManager inputManager;
         private DirectInput directInput;
+        private IEnumerable<CimInstance> allDevices;
+        private Regex regex;
```

Comment:
Consider renaming it to `xInputDeviceIdRegex`

Reply:
done